docs/messaging/messaging-spans.md

```diff
@@ -558,26 +560,26 @@ flowchart LR;
   linkStyle 0,1 color:green,stroke:green
 ```
 
-| Field or Attribute | Span Publish A | Span Publish B | Span Receive A B |
+| Field or Attribute | Span Send A | Span Send B | Span Receive A B |
 |-|-|-|-|
 | Span name | `send Q` | `send Q` | `poll Q` |
 | Parent |  |  |  |
-| Links |  |  | Span Publish A, Span Publish B |
-| Link attributes |  |  | Span Publish A: `messaging.message.id`: `"a1"`  |
-|                 |  |  | Span Publish B: `messaging.message.id`: `"a2"`  |
+| Links |  |  | Span Send A, Span Send B |
+| Link attributes |  |  | Span Send A: `messaging.message.id`: `"a1"`  |
+|                 |  |  | Span Send B: `messaging.message.id`: `"a2"`  |
 | SpanKind | `PRODUCER` | `PRODUCER` | `CONSUMER` |
 | `server.address` | `"ms"` | `"ms"` | `"ms"` |
 | `server.port` | `1234` | `1234` | `1234` |
 | `messaging.system` | `"kafka"` | `"kafka"` | `"kafka"` |
 | `messaging.destination.name` | `"Q"` | `"Q"` | `"Q"` |
 | `messaging.operation.name` | `"send"` | `"send"` | `"poll"` |
-| `messaging.operation.type` | `"publish"` | `"publish"` | `"receive"` |
+| `messaging.operation.type` | `"send"` | `"send"` | `"receive"` |
 | `messaging.message.id` | `"a1"` | `"a2"` | |
 | `messaging.batch.message_count` |  |  | 2 |
 
 ### Batch publishing
```

Comment:
```suggestion
### Batch sending
```

We also want to change here, right? The mermaid diagram + table are also using "publish" still.